### PR TITLE
fix(i18n): improve translations of Persian (Farsi)

### PR DESCRIPTION
**Pull request by Notorious-Ali**
## Summary of Changes

This update includes several improvements and translations to the Persian (Farsi) language file for better localization and user experience. Key changes are as follows:

### General Improvements:
- Added translations for previously untranslated strings such as **breadcrumb**, **carousel**, and **transfer** components.
- Enhanced user interaction prompts by providing better descriptions for controls like **colorpicker**, **datepicker**, and **slider**.
  
### New Translations:
- **Breadcrumb:** The term "Breadcrumb" was translated to "مسیر راهنما".
- **Carousel:** The navigation arrows and indicator were translated to:
  - "پیکان به جهت چپ" for the left arrow
  - "پیکان چرخان به جهت راست" for the right arrow
  - "سوئیچ چرخان به شاخص {index}" for the indicator
- **Transfer:** 
  - "فهرست ۱" and "فهرست ۲" for list titles
  - Translations for no checked and checked formats: 
    - "{total} آیتم" for no checked items
    - "{checked}/{total} انتخاب‌شده" for checked items.

### Detailed Control Descriptions:
- **Colorpicker:** Added accessible descriptions for users with screen readers:
  - Description of current color and instructions for selecting a new color.
  - Added translation for **alpha** control: "مقدار آلفا را انتخاب کنید".
- **Datepicker:** 
  - Added prompts for navigating the date, month, and year tables using keyboard arrows and the enter key.
  - Updated labels for day and month selection.
- **Slider:** Added descriptive labels for min and max values and range selection.
  
### UX Enhancements:
- Improved the clarity of instructions, such as in the **datepicker** component, guiding users to use keyboard arrows and enter key for date selection.
- Standardized phrases for consistency across different UI elements, such as replacing "حذف" with "پاک کردن" for clearing selections.

### CODE REVIEW

File: packages/locale/lang/fa.ts [hunk 1]
```diff
@@ -2,18 +2,27 @@ export default {
   name: 'fa',
   el: {
     breadcrumb: {
-      label: 'Breadcrumb', // to be translated
+      label: 'مسیر راهنما', // to be translated
```
Comment (plainheart) [inline packages/locale/lang/fa.ts:5]: Please remove the suffix comment `// to be translated` if it's translated.

**Review by plainheart — CHANGES_REQUESTED** [review]

File: packages/locale/lang/fa.ts (hunk 1 above)
Comment (Notorious-Ali) [reply packages/locale/lang/fa.ts]: done : https://github.com/element-plus/element-plus/pull/18290/commits/a4aa3291c9f7bb99d61aacd16df2b1aa6e0007ee

**Review by plainheart — APPROVED** [review]

**Comment by plainheart** [comment]
@Notorious-Ali Please fix the lint errors before we merge your PR. :)

**Comment by Notorious-Ali** [comment]
> @Notorious-Ali Please fix the lint errors before we merge your PR. :)

is it possible to translate  docs directly from github without crowdin?
its more easier to maintain

**Comment by plainheart** [comment]
@Notorious-Ali I'm afraid it's impossible for now. The translation of documentation can only be done in Crowdin.

**Pull request merged**